doc/level1.rst

```diff
@@ -980,4 +980,12 @@ Example Setup Message User Interaction
 
 .. todo::
 
-   TODO
+   Is fine just with this?
+
+.. this code decrypts the example Autocrypt Setup Message
+
+::
+
+    This is your Setup Code to decrypt the Autocrypt Setup Message:
```

Comment:
should this be "the Setup Code to decrypt your Autocrypt" (swapping "the" and "your")?  this makes it sound cleaner to me.